src/SFA.DAS.EAS.Account.Worker/Infrastructure/ServiceLocator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using StructureMap;
+
+namespace SFA.DAS.EAS.Account.Worker.Infrastructure
+{
+	public static class ServiceLocator
```

Comment:
I'm not quite sure how this would work. Specifically, I can't see how I would arrange for nested containers to be created. Will leave this for now and we can talk this through?